build/pxc-entrypoint.sh

```diff
@@ -385,7 +385,10 @@ if [ -z "$CLUSTER_JOIN" ] && [ "$1" = 'mysqld' -a -z "$wantHelp" ]; then
 			GRANT ALL ON *.* TO 'operator'@'${MYSQL_ROOT_HOST}' WITH GRANT OPTION ;
 
 			CREATE USER 'xtrabackup'@'%' IDENTIFIED BY '$(escape_special "${XTRABACKUP_PASSWORD}")';
-			GRANT ALL ON *.* TO 'xtrabackup'@'%';
+			GRANT BACKUP_ADMIN, RESOURCE_GROUP_USER, SERVICE_CONNECTION_ADMIN, PROCESS, RELOAD, LOCK TABLES, REPLICATION CLIENT ON *.* TO 'xtrabackup'@'%';
+			GRANT SELECT ON performance_schema.log_status TO 'xtrabackup'@'%';
+			GRANT SELECT ON performance_schema.keyring_component_status TO 'xtrabackup'@'%';
```

Comment:
keyring_component_status table was added only in 8.0.24, so it will not work e.g. for PXC 8.0.19 that is why you can see problems with smart update test

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/1398/commits/2e8ddc40bc499736f7608c19c839a8be6bbb9a3e